css/contact.css

```diff
@@ -0,0 +1,321 @@
+
+* {
+ padding: 0px;
+ margin: 0px;
+}
+
+body {
+ background: linear-gradient(to top,
+   #E6DACD 0%,
+   #E6DACD 90%,
+   #fff 90%,
+   #fff 100%);
+ width: 390px;
+ height: 844px;
+
+ align-content: center;
+ text-align: center;
+
+}
+
+/* header text */
+
+.square {
+ margin-left: 15px;
+ background-color: #0150FD;
+ width: 16px;
+ height: 16px;
+ margin-top: 10px;
+}
+
+.text_nav1 {
+ margin-left: 5px;
+ display: flex;
+ font-family: 'Frank Ruhl Libre', serif;
+}
+
+.text_nav2 {
+ margin-left: 16px;
+ font-family: 'Sarabun', sans-serif;
+}
+
+#toggle {
+ display: none;
+}
+
+
+/* Hamburger */
+.hamburger {
+ position: absolute;
+ top: 3em;
+ right: 5%;
+ margin-left: -2em;
+ margin-top: -45px;
+ width: 2em;
+ height: 45px;
+ z-index: 5;
+}
+
+.hamburger div {
+ position: relative;
+ width: 3em;
+ height: 7px;
+ border-radius: 3px;
+ background-color: rgb(74, 74, 248);
+ margin-top: 8px;
+ transition: all 0.3s ease-in-out;
+}
+
+/* Nav Styles */
+.nav {
+ position: fixed;
+ width: 100%;
+ height: 100%;
+ background-color: #F4ECE6;
+ top: -100%;
+ left: 0;
+ right: 0;
+ bottom: 0;
+ overflow: hidden;
+ transition: all 0.3s ease-in-out;
+ transform: scale(0);
+}
+
+.nav-wrapper {
+ position: relative;
+ overflow: hidden;
+ overflow-y: auto;
+ height: 100%;
+
+}
+
+nav {
+ text-align: left;
+ margin-left: 25%;
+
+}
+
+nav a {
+ position: relative;
+ text-decoration: none;
+ color: #020202;
+ font-size: 2em;
+ display: inline-block;
+ margin-top: 1.25em;
+ transition: color 0.2s ease-in-out;
+ letter-spacing: 1px;
+}
+
+nav a:before {
+ content: '';
+ height: 0;
+ position: absolute;
+ width: 0.25em;
+ background-color: #0150FD;
+ left: -0.5em;
+ transition: all 0.2s ease-in-out;
+}
+
+nav a:hover {
+ color: #0150FD;
+}
+
+nav a:hover:before {
+ height: 100%;
+}
+
+/**
+  Animations
+  **/
+.top-bun {
+
+ width: 20px;
+}
+
+.meat {
+ width: 40px;
+}
+
+.bottom-bun {
+ width: 20px;
+}
+
+#toggle:checked+.hamburger .top-bun {
+ transform: rotate(-45deg);
+ margin-top: 25px;
+}
+
+#toggle:checked+.hamburger .bottom-bun {
+ opacity: 0;
+ transform: rotate(45deg);
+}
+
+#toggle:checked+.hamburger .meat {
+ transform: rotate(45deg);
+ margin-top: -7px;
+}
+
+#toggle:checked+.hamburger+.nav {
+ top: 0;
+ transform: scale(1);
+}
+
+/* main */
+.h1_2 {
+ padding-left: 20px;
+ text-align: left;
+ font-weight: bold;
+ font-size: 40px;
+ font-family: 'Nanum Gothic', sans-serif;
+}
+
+.h2_1 {
+ padding-left: 20px;
+ text-align: left;
+ font-size: 20px;
+ font-family: 'Nanum Gothic', sans-serif;
+}
+
+p {
+ /* border: 2px solid black; */
+ padding-left: 20px;
+ text-align: left;
+ font-family: 'Sarabun', sans-serif;
+}
+
+/* grid */
+.container {
+ display: grid;
+ grid-template-columns: 1fr 1fr 1fr 1fr;
+ grid-template-rows: repeat(4, auto);
+ grid-template-areas:
+  "header header header header"
+  ". text_top text_top ."
+  ". form form ."
+  "footer footer footer footer"
+ ;
+}
+
+/* header */
```

Comment:
have you tried to install prettier or any other file formatter as adding an indent would be good in here